Prace_domowe/Praca_domowa4/Tomek_Nocoń/PD4.ipynb

```diff
@@ -0,0 +1,1432 @@
+{
```

Comment:
<p>Nie do końca o to chodziło, zmienną celu miała być kolumna <code>m2_price</code>  (problem regresji). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>